docs/user/tutorials/01-50-expose-and-secure-a-workload/01-55-ip-based-access-with-xff.md

```diff
@@ -0,0 +1,178 @@
+      ```json
+      {
+        "args": {
+          "show_env": "true"
+        },
+        "headers": {
+          "Accept": "...",
+          "Host": "...",
+          "User-Agent": "...",
+          "X-Envoy-Attempt-Count": "...",
+          "X-Envoy-External-Address": "165.1.187.197",
+          "X-Forwarded-Client-Cert": "...",
+          "X-Forwarded-For": "165.1.187.197",
+          "X-Forwarded-Proto": "...",
+          "X-Request-Id": "..."
+        },
+        "origin": "165.1.187.197",
+        "url": "..."
+      }
+      ``` 
+
+      >[!TIP]
+      > You can check your public IP address at https://api.ipify.org.
+
+3. To configure IP-based access to the exposed workload, create an AuthorizationPolicy resource.
+    1. Go to **Istio > Authorization Policies** and choose **Create**.
+    2. Add a selector to specify the workload for which access should be configured.
+    3. Add a rule with a **From** field.
+    4. In the **RemoteIpBlocks** field, specify the IP addresses that should be allowed access to the workload.
+    5. Choose **Create**.
+
+
+#### **kubectl**
+1. To use the XFF header, configure the number of trusted proxies in the Istio custom resource.
+   1. Run the following command:
+       ```bash
+       kubectl patch istios/default -n kyma-system --type merge -p '{"spec":{"config":{"numTrustedProxies": 1}}}'
+       ```
+       Due to the variety of network topologies, the Istio CR must specify the configuration property **numTrustedProxies**, so that the client IP address can be extracted correctly.
+
+   2. If you use a GCP or Azure cluster, run the following command to set the traffic policy to `Local`. If you use a different cloud service provider, skip this step.
+
+       ```bash
+       kubectl patch istios/default -n kyma-system --type merge -p '{"spec":{"config":{"gatewayExternalTrafficPolicy": "Local"}}}'
+       ```
+       >[!WARNING]
+       > For production Deployments, it is strongly recommended that you deploy Istio Ingress Gateway Pod to multiple nodes if you enable `externalTrafficPolicy : Local`. For more information, see [Network Load Balancer](https://istio.io/latest/docs/tasks/security/authorization/authz-ingress/#network).
+       >
+       >Default Istio installation profile configures **PodAntiAffinity** to ensure that Ingress Gateway Pods are evenly spread across all nodes and, if possible, across different zones. This guarantees that the above requirement is satisfied if your IngressGateway autoscaling configuration **minReplicas** is equal to or greater than the number of nodes. You can configure autoscaling options in the Istio CR using the field **spec.config.components.ingressGateway.k8s.hpaSpec.minReplicas**.
+       
+       >[!TIP]
+       > If you use a GCP or Azure cluster, you can find your load balancer's IP address in the field **status.loadBalancer.ingress of the ingress-gateway** Service.
```

Comment:
The message is the same as in line 42, but the formatting is different, I'm not sure which one is correct. PTAL